README.md

```diff
@@ -11,16 +11,18 @@ agent.
 ## Getting fissile
 
 ### Prerequisites
-Building fissile needs [Go 1.7] or higher and [Docker].
+Building fissile needs [Go 1.9] or higher and [Docker].
 
-[Go 1.7]: https://golang.org/doc/install
+[Go 1.9]: https://golang.org/doc/install
 [Docker]: https://www.docker.com
 
 ### Build procedure
 Fissile requires generated code using additional tools, and therefore isn't
 `go get`-able.
 
 ```
+$ make src                                      # make the directory src 
```

Comment:
What is command `make src` and `export $GOPATH=$PWD` exactly? Should it be `mkdir src` and `export GOPATH=$PWD`?

Reply:
@edwardstudy Sorry for the confusing, it's a typo here. I'll fix it.

Reply:
Yes, that's how I read it. Sorry, should have caught this before I merged it.